src/main/java/medicaldoctor/entities/User.java

```diff
@@ -85,7 +88,11 @@ public String getLastName() {
     public void setLastName(String lastName) {
         this.lastName = lastName;
     }
-
+    
+    public void setEmail(){
+        this.email = this.firstName.substring(0, 1) + this.lastName.substring(1, this.lastName.length());
```

Comment:
put this in a EmailGenerator class like the password and username, we dont want to hardcode this here.

Reply:
also make it an actual email :)